app/src/models/item.ts

```diff
@@ -46,6 +50,23 @@ export class ItemModel {
     // get custom label from manifest
     const parser = new Maniiifest(manifest);
     this.hasItems = manifest.items.length > 0 ? true : false;
+    const canvases = Array.from(parser.iterateManifestCanvas());
+    const canvasAnnotations = Array.from(
+      parser.iterateManifestCanvasAnnotation()
+    );
+    console.log("canvasAnnotations is: ", canvasAnnotations);
+    // only used for order print button
+    this.isImage =
+      this.hasItems &&
+      (this.contentType === "singe image" ||
+        this.contentType === "multiple images");
+    // this.isImage = !!canvasAnnotations[0] && canvasAnnotations[0].body.type === "Image";
+
+    this.imageIDs = this.hasItems
+      ? canvases.map((canvas) => {
+          return canvas.id.split("/")[5];
```

Comment:
Might be useful to add a quick comment with an example url to explain why we're getting the specific index